searchInUnknownSizeRSA.java

```diff
@@ -0,0 +1,34 @@
+// Time Complexity :O(log n)
```

Comment:
Can you explain your complexity more elaborately. Since there is one while loop for searching the range where target can lie , so even that would contribute to complexity